### PR TITLE
Integrate harvester-node-disk-manager chart

### PR DESCRIPTION
**Pull request by weihanglo**
**Problem:**

Part of multi-disk manager feature #747 

**Solution:**

Integrate [node-disk-manager](https://github.com/harvester/node-disk-manager) as a helm chart.

**Related Issue:**

- #747 
- harvester/node-disk-manager#1
- harvester/charts#17

**Test plan:**

TODO. Will fill this blank once we meet a consensus on implementation of node-disk-manager.

### CODE REVIEW

File: deploy/charts/harvester/values.yaml [hunk 1]
```diff
@@ -327,6 +327,20 @@ harvester-network-controller:
     # Overrides the image tag whose default is the chart appVersion.
     tag: master-head
 
+harvester-node-disk-manager:
+  ## Specify to install harvester network controller,
```
Comment (Jellyfrog) [inline deploy/charts/harvester/values.yaml]: disk manager?

Comment (weihanglo) [reply deploy/charts/harvester/values.yaml]: Nice catch. Thanks!

**Review by guangbochen — APPROVED** [review]
LGTM

File: deploy/charts/harvester/values.yaml [hunk 2]
```diff
@@ -327,6 +327,20 @@ harvester-network-controller:
...
+harvester-node-disk-manager:
+  ## Specify to install harvester node disk manager,
+  ## defaults to "false".
+  enabled: false
+
+  ## Specify where is longhorn installed.
+  longhornNamespace: longhorn-system
+
+  image:
+    repository: rancher/harvester-network-controller
```
Comment (gitlawr) [inline deploy/charts/harvester/values.yaml]: incorrect repository?

Comment (weihanglo) [reply deploy/charts/harvester/values.yaml]: fixed at d068b73

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**